docs/Overview.md

```diff
@@ -194,27 +207,22 @@ The IS-05 schemas `sender_transport_params_ndi.json` and `constraints_schema_sen
 ```json
 [
   {
-    "server_ip": "10.10.10.10",
-    "server_port": 5960,
+    "machine_name": "ndi-machine-name",
     "source_name": "ndi-sender-unique-name",
-    "group_name": "camera1"
+    "source_url" : "..."
   }
 ]
 ```
+#### machine_name
+The name of the sender node in the NDI domain. A node SHALL report the machine name being utilized by the NDI SDK. A controller SHOULD NOT specify a new `machine_name` but SHOULD use `auto` or the current `machine_name`. 
 
-**server_ip**:
-IP address hosting the NDI server (IP address of interface bound to the server). If the parameter is set to auto the Sender MUST establish for itself which interface it can use, based on its own internal configuration. A `null` value indicates that the Sender has not yet been configured.
-
-**server_port**: 
-Port for the NDI server. If the parameter is set to `auto` the Sender MUST establish for itself which port it can use, based on its own internal configuration.
+#### source_name
+The name of the NDI stream created by the Native NDI Sender in the NDI domain. This parameter SHALL NOT be concatenated with `machine_name` in the format `machine_name(source_name)`.
 
-**source_name**:
-The name of the NDI stream as declared by the NDI Sender.
+| Informative note: In the NDI domain, the `source_name` and `machine_name` are concatenatedin the format `machine_name(source_name)` when streams are discovered and connected, however in the `transport_params`, these properties are kept independent.
 
-**group_name**: 
-The NDI group of the source. `null` indicates the default group.
-
-Although the NDI Advanced SDK does allow NDI Native Devices to specify additional Sender transport parameters, these parameters and properties SHOULD NOT be exposed in NMOS.
+#### source_url 
+The URL of the sender as utilized by the NDI SDK. Sender MUST report this upon activation. The contents are proprietary to the NDI SDK and SHOULD NOT be interpreted. A controller SHOULD specify `auto` or the current value when setting this property. If a controller does not specify `source_url` when updating the transport parameters, the sender shall treat it as `auto`.
```

Comment:
I disagree with that requirement. The Sender shall be able to report the url before activation. The url is not the property of the NDI SDK, it is ndi:// ... and the device that implement an NDI Sender or Receiver is free to interpret it.